diff --git a/server/main.py b/server/main.py
index 605c885..a02753d 100644
--- a/server/main.py
+++ b/server/main.py
@@ -19,7 +19,8 @@ from routers import (
     email_router,
     static_router,
     prices_router,
-    admin_dashboard_router
+    admin_dashboard_router,
+    org_router
 )
 import uvicorn
 import logging
diff --git a/server/routers/__init__.py b/server/routers/__init__.py
index 137ce20..7bb3030 100644
--- a/server/routers/__init__.py
+++ b/server/routers/__init__.py
@@ -9,6 +9,7 @@ from routers.email import email_router
 from routers.static import static_router
 from routers.prices import prices_router
 from routers.admin_dashboard import admin_dashboard_router
+from routers.organization import org_router
 
 __all__ = [
     "imagen_router",
@@ -21,5 +22,6 @@ __all__ = [
     "stripe_router",
     "email_router",
     "prices_router",
-    "admin_dashboard_router"
+    "admin_dashboard_router",
+    "org_router"
 ]
